crates/voicevox_core_c_api/src/lib.rs

```diff
@@ -98,19 +128,28 @@ pub unsafe extern "C" fn voicevox_open_jtalk_rc_new(
     })())
 }
 
-/// 参照カウントで管理されたOpenJtalkを削除する
-/// @param [in] open_jtalk 参照カウントで管理されたOpenJtalk
+/// ::OpenJtalkRc をデストラクトする。
+///
+/// @param [in] open_jtalk デストラクト対象
+///
+/// ## Example
 ///
-/// # Safety
-/// @open_jtalk 有効な :OpenJtalkRc のポインタであること
+/// ```c
+/// voicevox_open_jtalk_rc_delete(open_jtalk);
+/// ```
+///
+/// ## Safety
+///
+/// - `open_jtalk`は ::voicevox_open_jtalk_rc_new で得たものでなければならず、また既にこの関数で解放されていてはいけない。
```

Comment:
「有効」とは何かというのはちゃんと説明したので、これでもいいかもしれない。

```suggestion
/// - `open_jtalk`は<a href="#voicevox-core-safety">読み込みについて有効</a>でなければならない。
```

Reply:
あ、ここ`_delete`でしたね。こっちはそのままでいいかも
(各メソッドについてを意図していました)

Reply:
まあたぶん不慣れな人にとってはだいぶ前者のがわかりやすいですね！
けどまあ、どちらでも良さそう。